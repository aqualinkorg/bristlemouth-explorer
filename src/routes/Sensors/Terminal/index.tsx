import { Paper, styled } from '@mui/material';

const PaperContainer = styled(Paper)(({ theme }) => ({
  height: 'calc(40vh - 12rem)',
  margin: '0.5rem 1rem 1rem 1rem',
  padding: '1rem',
  flexShrink: 0,
  borderRadius: theme.spacing(1),
}));

function Terminal() {
  return <PaperContainer>Terminal</PaperContainer>;
}

export default Terminal;
