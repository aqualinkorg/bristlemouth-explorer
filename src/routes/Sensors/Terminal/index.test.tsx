import { render } from '@testing-library/react';
import Terminal from '.';

it('renders as expected', () => {
  const { container } = render(<Terminal />);
  expect(container).toMatchSnapshot();
});
