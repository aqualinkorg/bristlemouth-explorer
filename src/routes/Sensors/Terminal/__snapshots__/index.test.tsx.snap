// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-69hhoj-MuiPaper-root"
  >
    Terminal
  </div>
</div>
`;
